.github/workflows/ci-linters.yml

```diff
@@ -0,0 +1,33 @@
+name: Run pre-commit
+
+on:
+  pull_request:
+  workflow_dispatch:
+
+
+jobs:
+  lint:
+    runs-on: ubuntu-24.04
+    steps:
+      - name: Checkout TileDB-SOMA
+        uses: actions/checkout@v4
+
+      - name: Set up Python
```

Comment:
is there no R lint in CI?

Reply:
C++ has one in the pre-commits (clang)

Reply:
lintr/styler (linting and formatting) produce code that is unreadable (IMO) and that I hate maintaing. John ran styler a while ago and then we went back and manually applied fixes to it, as I've not yet figured out how to customize either of the two